src/plugins/loan-channel-query-map.js

```diff
@@ -715,6 +739,11 @@ export default {
 					id: 151,
 					url: 'b-i-p-o-c-business',
 					queryParams: 'status=fundRaising&country=US&loanTags=51&distributionModel=direct',
+					flssLoanSearch: {
+						countryIsoCode: ['US'],
+						sectorId: [51],
```

Comment:
Is this intentional? The query param has `loanTags=51` but the FLSS has `sectorId=51`